cloudant-client/src/test/java/com/cloudant/tests/CloudantClientTests.java

```diff
@@ -521,4 +521,14 @@ public MockResponse dispatch(RecordedRequest request) throws InterruptedExceptio
         // test.
         c.getAllDbs();
     }
+
+    /**
+     * Asser that a {@code null} URL causes an IllegalArgumentException to be thrown.

```

Comment:
Asser -> Assert

Reply:
Fixed in 105a75c4ba2c25e6390a5590245a3dbe93baf604